app/models/training/video.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'content'
+
+module Training
+  class Video < Content
+    # @return [String]
+    def self.content_type_id
+      'video'
+    end
+
+    def page_type = 'video_page'
+    def video_title = title
```

Comment:
Same as question, looks like page_type is an actual attribute now.

Reply:
Again, responsibility for a different ticket we have on the board I was thinking

Reply:
Good spot, that wasn't my intention, caused by copy/pasting beween migration files to make sure they were all the same.

Reply:
Not sure what this is in reply to - but I'm guessing the earlier comment about page type?